ArduCopter/Parameters.cpp

```diff
@@ -404,7 +404,7 @@ const AP_Param::Info Copter::var_info[] = {
 
     // @Param: FS_EKF_THRESH
     // @DisplayName: EKF failsafe variance threshold
-    // @Description: Allows setting the maximum acceptable compass and velocity variance
+    // @Description: Allows setting the maximum acceptable compass, velocity, position and height variances. Used in arming check and EKF failsafe.
```

Comment:
As a side note, we shouldn't have the "Allows setting the" in the param description.   All parameters "allow setting" so it's unnecessary.